### PR TITLE
added 1x1 scissor experiment (--enable-scissor-test)

### PR DESCRIPTION
**Pull request by Kasi-Reddy**

### CODE REVIEW

File: gputop/gputop-gl.c [hunk 1]
```diff
@@ -178,9 +179,14 @@ struct array *gputop_gl_surfaces;
...
 bool gputop_gl_force_debug_ctx_enabled = false;
 
+atomic_bool gputop_gl_scissor_test_enabled;
+atomic_bool gputop_gl_texture_test_enabled;
+
 atomic_bool gputop_gl_monitoring_enabled;
 atomic_int gputop_gl_n_queries;
 
+
+
```
Comment (pnmanolova) [inline gputop/gputop-gl.c]: We don't really need this extra white space here.

File: gputop/gputop-gl.c [hunk 2]
```diff
@@ -1043,6 +1055,20 @@ gputop_glXSwapBuffers(Display *dpy, GLXDrawable drawable)
...
 
+    //Scissor test code
+    scissor_test = atomic_load(&gputop_gl_scissor_test_enabled);
+    if (scissor_test)
+    {
+	glClear(GL_COLOR_BUFFER_BIT); // change to pfn_glClear?
+	pfn_glScissor(0, 0, 1, 1);
+	if (!glIsEnabled(GL_SCISSOR_TEST))
+		pfn_glEnable(GL_SCISSOR_TEST);
+    } else
```
Comment (pnmanolova) [inline gputop/gputop-gl.c]: The else statement needs to start on a new line.

File: gputop/gputop-ui.c [hunk 3]
```diff
@@ -48,6 +48,8 @@
 #include "gputop-gl.h"
 #endif
 
+
+
```
Comment (pnmanolova) [inline gputop/gputop-ui.c]: No need for these extra lines.

**Comment by Kasi-Reddy** [comment]
Thanks for reviewing, will fix that up

**Pull request closed without merging**